.github/workflows/python-ci-full.yml

```diff
@@ -0,0 +1,36 @@
+name: TileDB-SOMA Python CI (Full)
+
+on:
+  push:
+    branches: [main]
+  release:
+    types: [published]
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        # TODO: restore Windows build once we have C++/libtiledbsoma integration supported there
+        os: [ubuntu-22.04, macos-12]
+        # os: [ubuntu-22.04, macos-12, windows-2019]
+        python-version: ['3.7', '3.8', '3.9', '3.10']
+        include:
+          - runs-on: ubuntu-22.04
+            cc: gcc-11
+            cxx: g++-11
+          - runs-on: macos-12
+            cc: gcc-11
+            cxx: g++-11
+    uses: single-cell-data/TileDB-SOMA/.github/workflows/python-ci-single.yml@main
```

Comment:
I believe you should be able to call this (and below) `./.github/...` so that it refers to the code in the current revision (i.e., the same way things work now).

Reply:
GHA complained when a version wasn't specified, but then the workflow file didn't yet exist on main.  Can't test "no version defaults to main" until we merge.  Let me merge this to main and then I can open a new PR to address this.

Reply:
oh, I see now; need to use relative path.  Will try that on _this_ branch now. Thx!

Reply:
yeah, the documentation could be a little clearer that the `./` on the beginning is what is important, and that `.github/...` and `./.github/...` are not the same thing